Description: Explicit linking with "xml2".
 Should fix error building with "-Wl,--as-needed".
Author: Damien Raude-Morvan <drazzib@debian.org>
Forwarded: no
Last-Update: 2011-09-04
--- a/src/nebula/SConstruct
+++ b/src/nebula/SConstruct
@@ -59,7 +59,8 @@
     'nebula_sql',
     'nebula_log',
     'nebula_xml',
-    'crypto'
+    'crypto',
+    'xml2'
 ])
 
 if not env.GetOption('clean'):
--- a/src/scheduler/src/sched/SConstruct
+++ b/src/scheduler/src/sched/SConstruct
@@ -39,6 +39,7 @@
     'nebula_core',
     'nebula_template',
     'crypto',
+    'xml2'
 ])
 
 if not sched_env.GetOption('clean'):
